task_01.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Program to convert temperature in Fahrenheight to Celsius."""
+
+
+
+ABSOLUTE_DIFFERENCE = 273.15
+def farenheit_to_kelvin(degrees):
```

Comment:
function name isn't spelled correctly, that's why the test result read: `'module' object has no attribute 'fahrenheit_to_celsius'`

Because your module didn't have a function with that name.